enterprise-suite/tests/smoketest

```diff
@@ -21,27 +21,37 @@ function print_debug() {
         echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
         echo "Describe $pod:"
         echo
-        kubectl -n lightbend describe pod "$pod"
+        kubectl -n $NAMESPACE describe pod "$pod"
         echo
         echo "Previous logs for $pod:"
         echo
-        kubectl -n lightbend logs -p --all-containers "$pod"
+        kubectl -n $NAMESPACE logs -p --all-containers "$pod"
         echo
         echo
         echo "Logs for $pod:"
         echo
-        kubectl -n lightbend logs --all-containers "$pod"
+        kubectl -n $NAMESPACE logs --all-containers "$pod"
         echo
     done
     echo
     echo "========================================"
     echo
-    echo -e "\ntravis_fold:end:debug${1}.logs\r"
 }
 
-print_debug "pre"
```

Comment:
oops

Reply:
I removed this on purpose to try and avoid nuking Travis - as our builds seem to die during this step. I'm not sure we need it anyways, as we log at the end.

Reply:
Heh.  I figured that was some cruft I accidentally left behind.  :-)